helix-wezterm.sh

```diff
@@ -155,6 +155,5 @@ if [ "$act" != "null" ]; then
     command=$(yq e ".actions.$action.extensions.$extension" "$config_file")
   fi
 
-  expanded_command=$(echo $command | envsubst '$WEZTERM_PANE,$basedir,$binary_output,$buffer_name,$cursor_line,$interface_name,$test_name,$session,$entry')
-  echo "$expanded_command\r" | $send_to_pane
+  eval echo "$command\r" | $send_to_pane
```

Comment:
Please remove `\r` as it is parsed by the shell before `eval`, causing the command to be turned into something like:

```sh
+ eval echo 'tig blame $buffer_name +$cursor_line\r'                                                                  ++ echo tig blame helix-wezterm.sh +160r
```

This results in an error:

```sh
$ tig blame helix-wezterm.sh +160r
tig: Invalid number of options to blame
```